external-crates/move/crates/move-analyzer/prettier-move/tests/control-flow/if_expression.move

```diff
@@ -0,0 +1,83 @@
+// options:
+// printWidth: 80
+
+module test::if_expression {
+    public fun basic() {
+        if (true) call_something();
+        if (false) call_something_else_call_something_else_call_something_else();
+        if (false) {
+            call_something_else();
+        } else {
+            another_call();
+        };
+
+        if (very_long_binding) {
+            call_something();
+        } else {
+            call_something_else();
+        };
+    }
+
+    public fun control_flow() {
+        if (true) return call_something();
+        if (true) return call_something_else_call_something_else_call_something_else();
+    }
+
+    public fun folding() {
+
+        if (true) call_something()
+        else if (false) call_something_else()
+        else call_something_otherwise();
+
+        if (true) {
+            call_something()
+        } else if (false) {
+            call_something_else()
+        } else {
+            call_something_otherwise()
+        };
+
+        // should keep as is, no additions;
+        if (true) { let a = b; };
+
+        if (very_very_long_if_condition)
+            very_very_long_if_condition > very_very_long_if_condition;
+
+        let a = if (true) {
+            call_something_else();
+        } else {
+            call_something_else();
+        };
+
+        if (
+            very_very_long_if_condition > very_very_long_if_condition ||
```

Comment:
Please add tests with more complicated if conditions, in particular parenthesized in a nested fashion